arglonbot/Program.cs

```diff
@@ -72,6 +72,19 @@ void Console_CancelKeyPress(object? sender, ConsoleCancelEventArgs e)
     cts.Cancel();
 }
 
+string Message() => Now() switch
+{
+    { Month: 12, Day: 24 } now => "Happy christmas eve 🎅🏻",
+    { Month: 12, Day: 25 } => "Happy christmas 🎅🏻",
```

Comment:
My friends, cast off the shackles of British oppression.
```suggestion
    { Month: 12, Day: 24 } now => "Merry christmas eve 🎅🏻",
    { Month: 12, Day: 25 } => "Merry christmas 🎅🏻",
```